Pokemon Player/src/Battle.cpp

```diff
@@ -33,33 +42,38 @@ void Battle::fight()
 	{
 		//Read the current state of the battle
 		readRam();
+
+		//Update bost lead's stats
+		updatePoke(0);
+		updatePoke(-1);
 		//Calculate the best action
 		determineAction();
+
+		cout << nextMove[0] << " " << nextMove[1] << endl;
 		//Take the action
 		if(nextMove[0] == 1)
```

Comment:
The significance of 1 is unclear. Could you replace it with a constant or an enum?